classes/core/PKPString.inc.php

```diff
@@ -444,7 +444,9 @@ static function mime_content_type($filename, $suggestedExtension = '') {
 		}
 
 		// Check ambiguous mimetypes against extension
-		$ext = array_pop(explode('.',$filename));
+		// The result of "explode" can't be passed directly to "array_pop" in PHP 7.
```

Comment:
I'm using PHP7 and haven't run into this -- are you getting a warning?

Reply:
We are getting this exception, which is causing the UI to freeze even if the action is completed.

```
PHP Notice:  Only variables should be passed by reference in C:\inetpub\wwwroot\ojs\lib\pkp\classes\core\PKPString.inc.php on line 447
```

Reply:
Ah, OK. Happy to have warnings like this fixed. FYI, it's not recommended to have warning/error messages displayed to the browser, and if they are, it'll probably explain why the UI is freezing when you encounter a cosmetic warning.

Reply:
Merged: https://github.com/pkp/pkp-lib/commit/97e468a306a403bcc1875342a487096a9cef588e